conf/kubernetes-shuffle-service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: spark-shuffle-service
+    version: 2.1.0
+  name: shuffle
+spec:
+  template:
+    metadata:
+      labels:
+        app: spark-shuffle-service
+    spec:
+      volumes:
+        - name: temp-volume
+          hostPath:
+            path: '/tmp'
```

Comment:
Updating with the result of the SIG meeting discussion. We agreed `/var/tmp` would be good default base dir for this. So maybe `/var/tmp/spark-shuffle-files-0`, `/var/tmp/spark-shuffle-files-1`, etc.